condacolab.py

```diff
@@ -198,14 +203,26 @@ def install_from_url(
 
     env = env or {}
     bin_path = f"{prefix}/bin"
+    pre_conda_contents = ""
+
+    if env:
+        pre_conda_contents = "".join(f'export {key}="{shlex.quote(value)}"\n' for key, value in env.items())
```

Comment:
We need either parenthesis or square brackets to avoid a SyntaxError in some python versions, iirc:

```suggestion
        pre_conda_contents = "".join([f'export {key}="{shlex.quote(value)}"\n' for key, value in env.items()])
```